app/src/main/java/se/inera/intyg/minaintyg/util/html/HTMLFactory.java

```diff
@@ -19,8 +21,23 @@ public static String tag(String tagName, String className, String value) {
     if (value == null || tagName == null || tagName.isEmpty()) {
       return "";
     }
+    final var text = convertLineSeparatorsIfPresent(value);
+    return startTag(tagName, className) + text + endTag(tagName);
+  }
+
+  private static String convertLineSeparatorsIfPresent(String value) {
```

Comment:
It is fine to keep as is... but it is a bit redundant to first check for LINE_SEPARATOR and then replace. The <string>.replace(...) will do just that by itself.